mycelo/loadbot/bot.go

```diff
@@ -130,7 +130,7 @@ func runTransaction(ctx context.Context, client *ethclient.Client, chainID *big.
 	abi := contract.AbiFor("StableToken")
 	stableToken := bind.NewBoundContract(env.MustProxyAddressFor("StableToken"), *abi, client)
 
-	transactor := bind.NewKeyedTransactor(txCfg.Acc.PrivateKey)
+	transactor, _ := bind.NewKeyedTransactorWithChainID(txCfg.Acc.PrivateKey, chainID)
```

Comment:
Is this part of the PR?

Reply:
Keeping this to support the load bot using the latest signer. It's a bit extraneous, but otherwise it'd be a single line PR.